tools/sync-diff-inspector.md

```diff
@@ -0,0 +1,214 @@
+---
+title: sync-diff-inspector 用户文档
+category: tools
+---
+
+## sync-diff-inspector 简介
+sync-diff-inspector 是一个用于校验 MySQL／TiDB 中两份数据是否一致的工具，该工具提供了修复数据的功能（适用于修复少量不一致的数据）。
+
+主要功能：
+* 对比表结构和数据
```

Comment:
This list displays well on GitHub, but does not work on PingCAP website. Please add a blank line before this line.